packages/react-components/lib/tab-panel.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import Box from '@material-ui/core/Box';
+
+export interface TabPanelProps {
+  children?: React.ReactNode;
+  index: number;
+  value: number;
+}
+
+export function TabPanel(props: TabPanelProps): JSX.Element {
```

Comment:
Can't we just use material's `TabPanel` it doesn't seem to be doing anything much different.

Reply:
yea, it's the same implementation I believe. Tab Panel is still 'under development' because it's in the lab dir (not core) of the material-ui library

Reply:
:+1: I see, looks like the material tutorial creates it's own `TabPanel` as well.